opennms-doc/guide-development/src/asciidoc/text/rest/ifservices.adoc

```diff
@@ -0,0 +1,39 @@
+
+==== Categories
+
+Obtain or modify the status of a set of monitored services based on a given search criteria, based on nodes, IP interfaces, Categories, or monitred services itself.
+
+Examples:
+
+* /ifservices?node.label=onms-prd-01
+* /ifservices?ipInterface.ipAddress=192.168.32.140
+* /ifservices?category.name=Production
+* /ifservices?status=A
+
+===== GETs (Reading Data)
+
+[options="header", cols="5,10"]
+|===
+| Resource        | Description
+| `/ifservices`   | Get all configured monitored services for the given search criteria.
+|===
+
+Example:
+
+Get the forced unmanaged services for the nodes that belong to the requisition named Servers:
+
+`curl -u admin:admin "http://localhost:8980/opennms/rest/ifservices?status=F&node.foreignSource=Servers"`
```

Comment:
Just for code highlighting and formatting consistency with prettify / coderay, can we use this:

[source, bash]
----
curl -u admin:admin "http://localhost:8980/opennms/rest/ifservices?status=F&node.foreignSource=Servers"
----